### ISSUE TITLE
Attribute Error when importing pytorch

### ISSUE THREAD

**Issue by xychen9459** [pytorch/pytorch#2094]
I installed pytorch from source code. After installation, I tried to import pytorch, but I got the following error message: (using ipython)

```
In [1]: import torch
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-c031d3dd82fc> in <module>()
----> 1 import torch

/home/kb539/.pyenv/versions/anaconda2/lib/python2.7/site-packages/torch/__init__.pyc in <module>()
    337 import torch.multiprocessing
    338 import torch.sparse
--> 339 _C._init_names(list(torch._tensor_classes) + list(torch._storage_classes))
    340 
    341 # attach docstrings to torch and tensor functions

AttributeError: 'module' object has no attribute '_init_names'
```
I don't know how to fix this problem.....

**Comment by apaszke** [comment]
you must have two versions installed and be mixing them. uninstall pytorch and try again

**Comment by soumith** [comment]
@xychen9459 try the commands I gave in this post: https://github.com/pytorch/pytorch/issues/2091#issuecomment-315227119

**Comment by xychen9459** [comment]
@apaszke @soumith  There are two different pytorch versions installed on my lab computer. I have removed both of them and reinstalled the latest one. Now the problem is solved. Thanks!

**Comment by kcarnold** [comment]
I got this error even after building from source once.

Helpful debugging for this problem:
`ldd ~/anaconda/lib/python3.6/site-packages/torch/_C*.so`

I found that it had linked against a bunch of Torch libs in `~/anaconda/lib`, probably from a prior conda install. I did `rm ~/anaconda/lib/libTH*` and it worked. Well, that is, after the same `ldd` also told me that it was trying to link against CUDA 8 and I'd actually wanted the 7.5 build :)
